src/AppInstallerCLICore/Workflows/WorkflowBase.cpp

```diff
@@ -524,16 +525,28 @@ namespace AppInstaller::CLI::Workflow
         }
     }
 
+    void VerifyPath::operator()(Execution::Context& context) const
+    {
+        std::filesystem::path path = Utility::ConvertToUTF16(context.Args.GetArg(m_arg));
+
+        if (!std::filesystem::exists(path))
+        {
+            context.Reporter.Error() << Resource::String::VerifyFileFailedNotExist << ' ' << path.u8string() << std::endl;
+            AICLI_TERMINATE_CONTEXT(HRESULT_FROM_WIN32(ERROR_PATH_NOT_FOUND));
+        }
+    }
+
     void GetManifestFromArg(Execution::Context& context)
     {
         Logging::Telemetry().LogIsManifestLocal(true);
 
         context <<
-            VerifyFile(Execution::Args::Type::Manifest) <<
+            VerifyPath(Execution::Args::Type::Manifest) <<

```

Comment:
>VerifyPath [](start = 12, length = 10)

This one does not seem like it should have changed, `winget install -m a_directory` will now work?

Reply:
It will work if the directory contains only one manifest(either split manifest files or a singleton inside). The input will be checked in ValidateInput method in manifest parsing

---
In reply to: [571315313](https://github.com/microsoft/winget-cli/pull/740#discussion_r571315313) [](ancestors = 571315313)